src/test/resources/features/edit_contact_information_form.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Edit contact information
+
+  Background:
+    Given I log in Sales Force with valid User credentials
+    @createContact @deleteContact
+    Scenario: I edit the contact information
+      When I navigate to contact page
+      And I search for the contact on the table
+      And I edit the contact information with the following information
+        | Tratamiento    |  Ms          |
+        | Nombre         |  Elyzabeth   |
+        | Apellidos      |  Rojas       |
+        | Telefono       |  60601020    |
+      Then the contact name should be updated with the new values
```

Comment:
I think this is for  API because we can match web elements,  but we can ask the trainer